### ISSUE TITLE
[Bug]: SetContentProtection does not work on specific Windows computers

### ISSUE THREAD

**Issue by qiaoao666** [electron/electron#39907]
### Preflight Checklist

- [X] I have read the [Contributing Guidelines](https://github.com/electron/electron/blob/main/CONTRIBUTING.md) for this project.
- [X] I agree to follow the [Code of Conduct](https://github.com/electron/electron/blob/main/CODE_OF_CONDUCT.md) that this project adheres to.
- [X] I have searched the [issue tracker](https://www.github.com/electron/electron/issues) for a bug report that matches the one I want to file, without success.

### Electron Version

8.5.5

### What operating system are you using?

Windows

### Operating System Version

Windows11 version 22000.2295

### What arch are you using?

x64

### Expected Behavior

On my computer (windows11), setContentProtection() only takes effect when the window is first created and will not take effect when modified and set later.

On more people's computers, multiple calls to setContentProtection() will take effect normally.

### Actual Behavior

On my computer (windows11), setContentProtection() only takes effect when the window is first created and will not take effect when modified and set later.

On more people's computers, multiple calls to setContentProtection() will take effect normally.

I hope to identify the cause and fix this issue.

### Additional Information

none

**Comment by VerteDinde** [comment]
Hey @qiaoao666 , Electron 8.5.5 has been end of life for some time - can you confirm this error reproduces on a newer, currently supported version of Electron (Electron 22 or higher)? If you have any repro steps you can shre, that would also be appreciated.

**Comment by codebytere** [comment]
We haven't gotten a response to our questions in our comment above. With only the information that is currently in the issue, we don't have enough information to take action. In this event, i'm going to go ahead and close this but can reopen should you follow up with more info!

**Comment by qiaoao666** [comment]
Although Electron 8.5.5, I tested it and found that this issue also occurs on higher versions of Electron. Currently, I have found that only the second time SetContentProtection is set on my computer, it will not take effect. Of course, I am not sure what caused this issue, and whether it is a problem with my computer environment, unrelated to the electronics, which cannot be determined.

Here is my test demo:
https://github.com/qiaoao666/screenshot-demo

**Comment by qiaoao666** [comment]
I tested it and found that this issue also occurs on higher versions of Electron. Currently, I have found that only the second time SetContentProtection is set on my computer, it will not take effect. Of course, I am not sure what caused this issue, and whether it is a problem with my computer environment, unrelated to the Electron, which cannot be determined.

Here is my test demo:
https://github.com/qiaoao666/screenshot-demo
